src/main/resources/static/css/style2.css

```
@@ -274,3 +280,183 @@ main{
+    padding: 20px;
+    box-shadow: 0 10px 25px rgba(0,0,0,0.2);
+    display: none;
+    z-index: 100;
+}
+
+.user-dropdown {
+    position: absolute;
+    top: 50px;
+    right: 0;
+
+    background: white;
```

Stylelint (17.9.0)

[error] Expected no empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)